multimodal/src/autogluon/multimodal/models/utils.py

```diff
@@ -543,3 +545,44 @@ def get_hf_config_and_model(checkpoint_name: str, pretrained: Optional[bool] = T
         model = AutoModel.from_config(config)
 
     return config, model
+
+
+def custom_sigmoid(output: Dict):
```

Comment:
How about to change the name to `apply_sigmoid`?

Reply:
Thanks! Updated.